src/components/KymaModules/KymaModulesCreate.js

```diff
@@ -30,14 +30,17 @@ import './KymaModulesCreate.scss';
 import { Spinner } from 'shared/components/Spinner/Spinner';
 import { isFormOpenState } from 'state/formOpenAtom';
 import { isResourceEditedState } from 'state/resourceEditedAtom';
+import { ManagedWarnings } from './ManagedWarnings';
+import { ChannelWarning } from './ChannelWarning';
+import { UnmanagedModuleInfo } from './UnmanagedModuleInfo';
 
 export default function KymaModulesCreate({ resource, ...props }) {
   const { t } = useTranslation();
   const [kymaResource, setKymaResource] = useState(cloneDeep(resource));
   const [initialResource] = useState(resource);
   const [initialUnchangedResource] = useState(cloneDeep(resource));
-  const [, setIsResourceEdited] = useRecoilState(isResourceEditedState);
-  const [, setIsFormOpen] = useRecoilState(isFormOpenState);
+  const setIsResourceEdited = useSetRecoilState(isResourceEditedState);
```

Comment:
Another suggestion - Recoil is not maintained (last commit 1y ago) and has memory leaks. They may not be dangerous for Busola, but it'd be better to replace it with e.g. [jotai](https://jotai.org/).

Reply:
I know and want to change that, but we currently don't have the capacity to do this. It will be planned for the future.